plugin/trino-hive/src/main/java/io/trino/plugin/hive/security/SqlStandardAccessControl.java

```diff
@@ -243,6 +243,15 @@ public void checkCanShowColumns(ConnectorSecurityContext context, SchemaTableNam
         return columns;
     }
 
+    @Override
+    public Set<String> filterTableSchema(ConnectorSecurityContext context, SchemaTableName tableName, Set<String> columns)
+    {
+        if (!hasAnyTablePermission(context, tableName)) {
```

Comment:
I don't want to change behavior of any existing access control, so please return `columns` here. Also this is not correct implementation for `SqlStandard`